lib/helpers/calculator_helpers.js

```diff
@@ -132,6 +132,9 @@ module.exports = class CalculatorHelpers {
       if ('DENEXCEP' in populationResults) {
         populationResultsHandled.DENEXCEP = 0;
       }
+      if ('observation_values' in populationResults) {
+        populationResultsHandled.observation_values = [];
+      }
```

Comment:
do we need to conder multiple IPP scenario? I believe the ratio measures can have multiple IPPs.

Reply:
Yes, multiple IPP scenarios do need to be considered.

The "createEpisodePopulationValues" method calls back to the "handlePopulationValues" method for each Episode of Care.  The fix above should address the results for each episode as well as the patient totals.

https://github.com/projecttacoma/cqm-execution/blob/9ce4a73874194157990ca088209906d19d7889c8/lib/helpers/calculator_helpers.js#L305

Reply:
thanks for the clarification and fix!